presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -342,6 +347,15 @@ private static void validateColumnStatistics(SchemaTableName table, String parti
         });
     }
 
+    private static void checkMinusOneNullsCountStatistic(boolean expression, SchemaTableName table, String partition, String column, String message, Object... args)
+    {
+        if (!expression) {
```

Comment:
if `nullsCount = -1`, we throw PrestoException with specific error code `HIVE_NULL_COLUMN_NEGATIVE_STATISTICS`